packages/create-react-native-library/templates/common/CONTRIBUTING.md

```diff
@@ -22,6 +22,9 @@ yarn
 The [example app](/example/) demonstrates usage of the library. You need to run it to test any changes you make.
 
 It is configured to use the local version of the library, so any changes you make to the library's source code will be reflected in the example app. Changes to the library's JavaScript code will be reflected in the example app without a rebuild, but native code changes will require a rebuild of the example app.
+<% if (project.nitro) { -%>
```

Comment:
Does this add newline between the paragraphs? 🤔

Reply:
added it